### DISCUSSION TITLE
Welcome to Pengwin Discussions!

### DISCUSSION THREAD

**Discussion by crramirez** [WhitewaterFoundry/Pengwin#659] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mattmazer0397** [comment]
Being asked to talk about myself is my worst nightmare (haha). On an actual note, I'm just a tech guy. I got my start in cyber security, and I'm now a blend of that and systems administration. In my free time, I'm pursuing a degree in software development, and that's where I make use of Pengwin/WSL2.

I find the technology behind WSL fascinating, and I chose Pengwin over the other distros because it caught my attention the most...though I do enjoy the responsive support community, and the fact that it was built from the ground up for WSL.

If anyone is interested about what I'm working on, I'm currently using Python to build a scoreboard for Phase 10 (a card game my family loves). When it comes to college though, I'm still undecided between C# and Java track. I'll be returning to Western Governors University in the spring, and those are the courses they offer. I hope to one day contribute regularly to projects ton Github - maybe even Pengwin - but only time will tell.

**Reply by crramirez** [reply]
Hello @mattmazer0397 ,

Very good introduction. What do you use for Python? penv? poetry? What IDE do you use? PyCharm VS Code?

C# is getting a lot of traction nowadays, but Java is still well-positioned in the enterprise. The languages are pretty similar and ASP.NET and Spring Boot are similar also. In my opinion, Java is easier to learn (fewer constructions and tricks), C# has more characteristics therefore more to learn. 

BTW, pengwin-setup has .NET and Java installers if you want to learn on the Linux side.

Your contributions of course will be welcomed.

Regards
